src/main/java/pacman/food.java

```diff
@@ -4,4 +4,8 @@ public food (int x, int y,boolean exist){
 		super(x,y,exist);
         this.special = false;
```

Comment:
You need to organize your code and remove blank spaces

Reply:
i couldn't find this file 
i think it doesn't exists in this branch